staging/src/k8s.io/api/admissionregistration/v1beta1/webhook_config_interface.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"k8s.io/apimachinery/pkg/util/json"
+)
+
+// Implementing k8s.io/apiserver/pkg/util/webhook.ClientConfigInterface
+
+func (c *WebhookClientConfig) GetURL() *string {
```

Comment:
see comments on conversion PR: this belongs where the interface is used. This is not API and should not live on the API types.